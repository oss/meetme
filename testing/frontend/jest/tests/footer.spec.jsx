import renderer from 'react-test-renderer';
import Footer from '../../../../frontend/components/footer.jsx'

//good example of a simple snapshot test
it('should render as expected and match the snapshot', () => {
    const container = renderer.create(<Footer />);
    expect(container).toMatchSnapshot();
}); 