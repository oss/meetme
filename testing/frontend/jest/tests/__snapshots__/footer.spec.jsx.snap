// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render as expected and match the snapshot 1`] = `
<div
  class="flex justify-center p-5 text-sm text-white bg-gray-700"
>
  <div
    class="w-3/5 text-center"
  >
    Rutgers is an equal access/equal opportunity institution. Individuals with disabilities are encouraged to direct suggestions, comments, or complaints concerning any accessibility issues with Rutgers websites to
    <a
      class="text-gray-300 hover:underline"
      href="mailto:accessibility@rutgers.edu"
    >
       accessibility@rutgers.edu 
    </a>
    or complete the
     
    <a
      class="text-gray-300 hover:underline"
      href="https://rutgers.ca1.qualtrics.com/jfe/form/SV_57iH6Rfeocz51z0"
    >
      Report Accessibility Barrier / Provide Feedback Form.
    </a>
    <br />
    <a
      class="text-gray-300 hover:underline"
      href="https://www.rutgers.edu/copyright-information"
      rel="nofollow"
      target="_blank"
    >
      Copyright Â©2024
    </a>
     
    <a
      class="text-gray-300 hover:underline"
      href="https://www.rutgers.edu"
    >
      Rutgers, The State University of New Jersey
    </a>
    . All rights reserved.
  </div>
</div>
`;
